src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets

```diff
@@ -144,6 +145,37 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CallTarget Targets="$(TargetsTriggeredByCompilation)" Condition="'$(TargetsTriggeredByCompilation)' != ''"/>
   </Target>
 
+  <Target Name="GenerateVersionFile">
```

Comment:
IL SDK is used by number of other people, not just by src/libraries. Is this target going to break them ?

Reply:
It should not break anyone, but to be safe i can add https://github.com/dotnet/runtime/pull/39675/files#diff-aaab00d0cd2d34f5500205e60143a555R16 only in the libraries ilproj which will make ensure that it runs only for that ilproj